### PR TITLE
Add classifiers and package data to setup.py

### PR DESCRIPTION
**Pull request by morenol**
Move readme to .rst and add initial configuration of setup.py that we need in order to  deploy releases to pypi

### CODE REVIEW

File: {{ cookiecutter.project_slug }}/setup.py [hunk 1]
```diff
@@ -56,21 +56,41 @@ def get_version(*file_paths):
     raise RuntimeError('Unable to find version string.')
 
 
+with open("README.rst", "r") as fh:
+    README = fh.read()
```
Comment (felipemontoya) [inline {{ cookiecutter.project_slug }}/setup.py:60]: This would be the complete file, right?

File: {{ cookiecutter.project_slug }}/README.rst [hunk 2]
```diff
@@ -1,22 +1,29 @@
-# {{ cookiecutter.project_desc }}
+===============================
+{{ cookiecutter.project_desc }}
+==============================
```
Comment (felipemontoya) [inline {{ cookiecutter.project_slug }}/README.rst]: Probably we will have to fix this for every project_desc lenght, but at least on the template, they should match the lenght from before and after

**Review by felipemontoya — COMMENTED** [review]
Two minor questions inline

File: {{ cookiecutter.project_slug }}/setup.py (hunk 1 above)
Comment (morenol) [reply {{ cookiecutter.project_slug }}/setup.py:60]: Yes

File: {{ cookiecutter.project_slug }}/README.rst (hunk 2 above)
Comment (morenol) [reply {{ cookiecutter.project_slug }}/README.rst]: Good catch,
I think that we can do something like:

```
{{ "=" * ( cookiecutter.project_desc | count) }}
{{ cookiecutter.project_desc }}
{{ "=" * ( cookiecutter.project_desc | count) }}
```

Comment (morenol) [reply {{ cookiecutter.project_slug }}/README.rst]: Changed and tested locally, it works

**Review by felipemontoya — APPROVED** [review]

**Pull request merged**